.gitattributes

```diff
@@ -0,0 +1,3 @@
+* text=auto eol=lf
+*.png binary
```

Comment:
does git try to handle some png as text files?

Reply:
No, it's smart enough without it:
<pre>
i/-text w/-text attr/                   docs/source/_static/product_logo_32.png
i/-text w/-text attr/                   docs/source/_static/product_logo_name_48.png
</pre>
Just added it as a precaution, but not really needed.

Reply:
remove this line then maybe?